src/libServer/LookupServer.h

```diff
@@ -19,18 +19,17 @@
 #define ZILLIQA_SRC_LIBSERVER_LOOKUPSERVER_H_
 
 #include "Server.h"
-#include "common/Constants.h"
-#include "libCrypto/EthCrypto.h"
-#include "libEth/Eth.h"
-#include "libUtils/GasConv.h"
-#include "libUtils/Logger.h"
+#include "EthLookupServer.h"
+
+
 
 class Mediator;
 
 typedef std::function<bool(const Transaction& tx, uint32_t shardId)>
     CreateTransactionTargetFunc;
 
 class LookupServer : public Server,
+                     public EthLookupServer,
```

Comment:
This way of inheriting could surely confuse future readers.
A LookupServer is certainly not a kind of EthLookupServer.  It is the other way around, and  I think if you swap the inheritance line, this PR should also "just work" (well, maybe ..)

Reply:
I just msgd you, the inheritance isn't important, I think composition is easy now the way things operate.

Reply:
Upon reflection the name EthLookupServer was a bad choice, the class is in fact the owner of the additional RPC methods that should be added to LookupServer when EVM is enabled.

The class is now renamed to EthRPCMethods and I think makes more sense.

The inheritance provides the exact funtionality required as if the methods were inlined in the lookupserver class, no changes to any existing users of LokkupServer were required the Test Mock routines and IsolatedServer that utilises the code directly is unaware the shape of the class has changed.